README.md

```diff
@@ -1,6 +1,7 @@
 [![Logo][jhipster-image]][jhipster-url]
 
-[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Downloads][npmcharts-image]][npmcharts-url]
+[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter](https://badges.gitter.im/jhipster/generator-jhipster.svg)](https://gitter.im/jhipster/generator-jhipster?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
Can you do the same as previously done with the URL? I mean:
  - like that `[![NPM version][npm-image]][npm-url]`

See the bottom of the file to find the "aliases"

Reply:
Ops, just copied and pasted without checking. :smile: Done.